File: AR7ExportToolbar/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.IO;
using System.Text;
using System.Windows.Forms;
using AR7ExportToolbar.ToolStripExport;

namespace AR7ExportToolbar
{
	public partial class Form1 : Form
	{
		public Form1()
		{
			InitializeComponent();

			ExportToolStripButton button = new ExportToolStripButton(viewer1);
			viewer1.Toolbar.MainBar.Items.Insert(3, button);
		}

		private void viewer1_Load(object sender, EventArgs e)
		{
			viewer1.LoadDocument("PageReport1.rdlx");
		}
	}
}

File: AR7ExportToolbarVB/Form1.vb
Public Class Form1
	Public Sub New()

		' This call is required by the designer.
		InitializeComponent()

		' Add any initialization after the InitializeComponent() call.
		Dim button As ExportToolStripButton = New ExportToolStripButton(Viewer1)
		' button.AvailableExports.Remove("PDF") ' remove exports that you don't want to make available, even if the assembly is there

		Viewer1.Toolbar.MainBar.Items.Insert(3, button)
	End Sub

	Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
		Viewer1.LoadDocument("PageReport1.rdlx")
	End Sub
End Class
